Задание три/quest_7.py

```diff
@@ -0,0 +1,19 @@
+def Print_Matrix(matrix):
+    for i in range(len(matrix)):
+        for j in range(len(matrix[i])):
+            print(matrix[i][j], end = ' ')
+
+
+def Calc_Matrix(matrix):
+    for i in range(len(matrix)):
+        for j in range(len(matrix[i])):
+            if i==j:
+                print(matrix[i][j]*3, end = ' ')
+            else:
+                print(matrix[i][j])
```

Comment:
def Calc_Matrix(matrix):
___for i in range(len(matrix)):
______for j in range(len(matrix[i])):
_________if i==j:
____________matrix[i][j]*=3
_________print(matrix[i][j], end = ' ')
______print()